lib/pages/Terms.dart

```diff
@@ -1,113 +1,90 @@
+import 'package:carriage_rider/widgets/Buttons.dart';
 import 'package:flutter/material.dart';
 import 'package:carriage_rider/pages/Home.dart';
+import 'package:url_launcher/url_launcher.dart';
 
 class TermsOfService extends StatelessWidget {
   Widget acceptButton(BuildContext context) {
-    return Align(
-        alignment: Alignment.bottomCenter,
-        child: Padding(
-          padding: const EdgeInsets.only(bottom: 20.0),
-          child: ButtonTheme(
-              minWidth: MediaQuery.of(context).size.width * 0.8,
-              height: 50.0,
-              shape: RoundedRectangleBorder(
-                  borderRadius: BorderRadius.circular(10)),
-              child: RaisedButton(
-                  onPressed: () {
-                    Navigator.push(context,
-                        new MaterialPageRoute(builder: (context) => Home()));
-                  },
-                  elevation: 2.0,
-                  color: Colors.black,
-                  textColor: Colors.white,
-                  child: Text('Accept All',
-                      style: TextStyle(
-                          fontWeight: FontWeight.bold, fontSize: 17)))),
-        ));
+    return CButton(
```

Comment:
Good catch, thanks! I made this page scrollable to be responsive for large text, so I'll change it so that the button is sticky at the bottom like we did for the ride flow pages